tests/test_mlkit.py

```diff
@@ -33,27 +35,84 @@
     'displayName': DISPLAY_NAME_1
 }
 MODEL_1 = mlkit.Model(MODEL_JSON_1)
-_DEFAULT_GET_RESPONSE = json.dumps(MODEL_JSON_1)
+
+MODEL_ID_2 = 'modelId2'
+MODEL_NAME_2 = 'projects/{0}/models/{1}'.format(PROJECT_ID, MODEL_ID_2)
+DISPLAY_NAME_2 = 'displayName2'
+MODEL_JSON_2 = {
+    'name': MODEL_NAME_2,
+    'displayName': DISPLAY_NAME_2
+}
+MODEL_2 = mlkit.Model(MODEL_JSON_2)
+
+MODEL_ID_3 = 'modelId3'
+MODEL_NAME_3 = 'projects/{0}/models/{1}'.format(PROJECT_ID, MODEL_ID_3)
+DISPLAY_NAME_3 = 'displayName3'
+MODEL_JSON_3 = {
+    'name': MODEL_NAME_3,
+    'displayName': DISPLAY_NAME_3
+}
+MODEL_3 = mlkit.Model(MODEL_JSON_3)
+
 _EMPTY_RESPONSE = json.dumps({})
+_DEFAULT_GET_RESPONSE = json.dumps(MODEL_JSON_1)
```

Comment:
Nit: I see some constants prefixed with `_` and some not. Should we be consistent within the same test file?

Reply:
done